CONTRIBUTING.md

```diff
@@ -22,6 +22,10 @@ To get started, go ahead and fork this repo. Once you've done that, there are a
 
 `npm run build`
 
+## Testing
```

Comment:
I feel like we should add more context to the title, or maybe first sentence, to say this is so you can test local changes in your app.

Reply:
Thanks for catching this, that's a great point. I have added more context to the docs in 08cb441